### PR TITLE
Lavaland Ruin

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
A biodome filled with snow

### CODE REVIEW

**Comment by KazeEspada** [comment]
MUH TREE CUTTING!

**Comment by KorPhaeron** [comment]
No clue why it isn't compiling, does so locally

**Comment by KazeEspada** [comment]
https://travis-ci.org/tgstation/-tg-station/jobs/112785090#L279

File: _maps/templates/lavaland_surface_biodome_winter.dmm [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+"t" = (/obj/structure/table/wood,/turf/simulated/floor/wood{baseturf = /turf/simulated/floor/plating/lava/smooth; name = "floor"},/area/ruin/powered)
+"u" = (/obj/structure/chair/comfy/brown{tag = "icon-comfychair (WEST)"; icon_state = "comfychair"; dir = 8},/turf/simulated/floor/wood{baseturf = /turf/simulated/floor/plating/lava/smooth; name = "floor"},/area/ruin/powered)
+"v" = (/turf/simulated/wall/mineral/wood{baseturf = /turf/simulated/floor/plating/lava/smooth},/area/ruin/powered)
+"w" = (/obj/structure/displaycase/captain,/turf/simulated/floor/wood{baseturf = /turf/simulated/floor/plating/lava/smooth; name = "floor"},/area/ruin/powered)
+"x" = (/obj/structure/bed/dogbed,/turf/simulated/floor/wood{baseturf = /turf/simulated/floor/plating/lava/smooth; name = "floor"},/area/ruin/powered)
+"y" = (/obj/structure/flora/tree/pine/xmas,/turf/simulated/floor/plating/asteroid/snow{baseturf = /turf/simulated/floor/plating/lava/smooth},/area/ruin/powered)
+"z" = (/obj/structure/flora/grass/brown{step_x = -1},/turf/simulated/floor/plating/asteroid/snow{baseturf = /turf/simulated/floor/plating/lava/smooth},/area/ruin/powered)
```
Comment (Jordie0608) [inline _maps/templates/lavaland_surface_biodome_winter.dmm]: >{step_x = -1}

Kor comeon

Comment (KorPhaeron) [inline _maps/templates/lavaland_surface_biodome_winter.dmm]: How does this happen? I didnt edit that nor has it ever happened to me before

Comment (RemieRichards) [inline _maps/templates/lavaland_surface_biodome_winter.dmm]: did you use the Nudge button?

Comment (KorPhaeron) [inline _maps/templates/lavaland_surface_biodome_winter.dmm]: Not that I'm. Accidentally hitting a key at some point sounds likely though

**Pull request merged**